javascript/preflight.js

```diff
@@ -0,0 +1,46 @@
+function runPreFlightChecks() {
+  const isHttp      = window.location.protocol == 'http:',
+        isLocalhost = window.location.hostname == 'localhost'
+        isUnsafe    = !(isHttp && isLocalhost);
+
+  if (isUnsafe) {
+    const isHttpMessage = [
+      'This page is running over the web over HTTP.',
+      'You should use HTTP and rotate your access token.',
```

Comment:
Shouldn't this be "You should use HTTPS"?

Reply:
Correct, amended.